webpack.prod.js

```diff
@@ -19,15 +19,16 @@ module.exports = merge(common, {
   },
   plugins: [
     new MiniCssExtractPlugin({
-      filename: '[name].css',
-      chunkFilename: '[id].css'
+      filename: "[name].css",
+      chunkFilename: "[id].css"
     })
   ],
   module: {
     rules: [
       {
         test: /\.css$/,
-        use: [MiniCssExtractPlugin.loader, 'css-loader']
+        use: [MiniCssExtractPlugin.loader, "css-loader"],
+        sideEffects: true
```

Comment:
Had to add this to get imports to work @seanforyou23 

 Should we file an issue for react-patternfly to fix the react-core/package.json side-effects: false  to be side-effects:["*.css"] . ?